changesummary.py

```diff
@@ -0,0 +1,268 @@
+from enum import IntEnum
+import json
+from multiprocessing import Pool
+import os
+import pandas as pd
+import numpy as np
```

Comment:
I would be tempted to pull `changessummary.py`, and `create_commits.sh` into a `scripts/` folder.

Reply:
Done.